demo/chbench/mzworkflows.py

```diff
@@ -0,0 +1,92 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from typing import List
+
+import requests
+
+from materialize.mzcompose import PrometheusSQLExporter, Workflow
+
+
+def workflow_demo(w: Workflow, args: List[str]):
+    """Run CH-benCHmark without any load on Materialize"""
+
+    # Start Materialize.
+    w.start_services(services=["materialized"])
+    w.wait_for_mz()
+
+    # Start MySQL and Debezium.
+    w.start_services(services=["mysql", "connect"])
+    w.wait_for_tcp(host="mysql", port=3306)
+    w.wait_for_tcp(host="connect", port=8083)
+
+    # Generate initial data.
+    w.run_service(
+        service="chbench",
+        command="gen --config-file-path=/etc/chbenchmark/mz-default-mysql.cfg --warehouses=1",
+    )
+
+    # Run load generator.
+    w.run_service(
+        service="chbench",
+        command=[
+            "run",
+            "--config-file-path=/etc/chbenchmark/mz-default-mysql.cfg",
+            "--dsn=mysql",
+            "--gen-dir=/var/lib/mysql-files",
+            "--analytic-threads=0",
+            "--transactional-threads=1",
+            "--run-seconds=86400",
+            "--mz-sources",
+            *args,
+        ],
+        daemon=True,
+    )
+
+    # Start Debezium.
+    connect_port = w.composition.find_host_ports("connect")[0]
+    response = requests.post(
+        f"http://localhost:{connect_port}/connectors",
+        json={
+            "name": "mysql-connector",
+            "config": {
+                "connector.class": "io.debezium.connector.mysql.MySqlConnector",
+                "database.hostname": "mysql",
+                "database.port": "3306",
+                "database.user": "debezium",
+                "database.password": "debezium",
+                "database.server.name": "debezium",
+                "database.server.id": "1234",
+                "database.history.kafka.bootstrap.servers": "kafka:9092",
+                "database.history.kafka.topic": "mysql-history",
+                "database.allowPublicKeyRetrieval": "true",
+                "time.precision.mode": "connect",
+            },
+        },
+    )
+    # Don't error if the connector already exists.
+    if response.status_code != requests.codes.conflict:
+        response.raise_for_status()
+
+
+def workflow_load_test(w: Workflow):
+    """Run CH-benCHmark with a selected amount of load against Materialize."""
+    w.start_services(services=["prometheus-sql-exporter"])
+    workflow_demo(
+        w,
+        [
+            "--peek-conns=1",
+            "--mz-views=q01,q02,q05,q06,q08,q09,q12,q14,q17,q19",
```

Comment:
where is `mz-views` defined?

Reply:
https://github.com/MaterializeInc/materialize/blob/32f3600a714d669fbeb3ed3c808deebf82efb201/demo/chbench/chbench/src/chBenchmark.cc#L377

https://github.com/MaterializeInc/materialize/blob/32f3600a714d669fbeb3ed3c808deebf82efb201/demo/chbench/chbench/src/chBenchmark.cc#L624-L641

https://github.com/MaterializeInc/materialize/blob/main/demo/chbench/chbench/mz-default-mysql.cfg

Reply:
oh yeah I see.